docs/source/configuration.rst

```diff
@@ -429,6 +431,15 @@ And here is an example of server-based authentication using SASL::
     auth_password = SopelIsGreat!  # your bot's password
     auth_target = PLAIN            # default sasl mechanism
 
+Example of authentication to a ZNC bouncer::
+
+    [core]
+    auth_method = server           # select server-based auth
+    # auth_username can't be used with server authentication, so instead
+    # we combine the ZNC username, network name, and password here:
+    auth_password = Sopel/libera:SopelIsGreat!
+    # Don't forget to configure your ZNC to log in to the real network!
```

Comment:
Alright, it seems my comment wasn't clear: this part should not be within the example, as it may imply that it is expected to put more configuration *within the same file*:

```suggestion

Don't forget to configure your ZNC to log in to the real network!
```